.github/workflows/build-push-via-cicd-actions.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Push Image #by triggering cicd-actions workflow via Github API Call
+
+on:
+  push:
+    branches: [ feature/custatevec, feature/ZCLOUD-260/add-ci-with-github-actions ]
+  
+jobs:
+  trigger-build-and-push:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Trigger cicd-actions repository workflow over Github API
+        run: |
+              curl \
```

Comment:
this action could feasibly be successful, because the network call was good, but the build could potentially fail right? is there a way to know when those actions failed and to view the logs?

Reply:
Yes, this is only a trigger to central build repository: [click](https://github.com/zapatacomputing/cicd-actions/actions). where the actual build is done and can be viewed by anyone